app/template_statistics/rest.py

```diff
@@ -70,31 +58,45 @@ def get_template_statistics_for_template_id(service_id, template_id):
     return jsonify(data=data)
 
 
-def get_template_statistics_for_7_days(limit_days, service_id):
-    cache_key = cache_key_for_service_template_counter(service_id)
-    template_stats_by_id = redis_store.get_all_from_hash(cache_key)
-    if not template_stats_by_id:
-        stats = dao_get_template_usage(service_id, limit_days=limit_days)
-        cache_values = dict([(x.template_id, x.count) for x in stats])
-        if cache_values:
-            redis_store.set_hash_and_expire(cache_key,
-                                            cache_values,
-                                            current_app.config['EXPIRE_CACHE_TEN_MINUTES'])
-    else:
-        stats = dao_get_templates_for_cache(template_stats_by_id.items())
-    return stats
-
-    # TODO: can only switch to this code when redis has been populated (either through time passing or a manual step)
-    # from collections import Counter
-    # from notifications_utils.redis_client import RedisException
-    # template_stats_by_id = Counter()
-    # for day in last_7_days:
-    #     # "<SERVICE_ID>-template-usage-{YYYY-MM-DD}"
-    #     key = cache_key_for_service_templates_used_per_day(service_id, limit_days)
-    #     try:
-    #         template_stats_by_id += Counter(redis_store.get_all_from_hash(key, raise_exception=True))
-    #     except RedisException:
-    #         # TODO: ????
-    #
-    # # TODO: streamline db query and avoid weird unions if possible.
-    # return dao_get_templates_for_cache(template_stats_by_id.items())
+def get_template_statistics_for_last_n_days(service_id, limit_days):
```

Comment:
No tests for this method?

Reply:
i viewed them as being part of of the `get_template_statistics_for_service_by_day` tests